upup/models/cloudup/resources/addons/eks-pod-identity-webhook.addons.k8s.io/k8s-1.16.yaml.template

```diff
@@ -80,7 +80,7 @@ spec:
       serviceAccountName: pod-identity-webhook
       containers:
       - name: pod-identity-webhook
-        image: amazon/amazon-eks-pod-identity-webhook:v0.4.0
+        image: amazon/amazon-eks-pod-identity-webhook:v0.5.3
```

Comment:
Why not `v0.5.7`?
https://github.com/aws/amazon-eks-pod-identity-webhook/releases/tag/v0.5.7

Reply:
I would prefer the latest version in this case. Thanks!